### DISCUSSION TITLE
annoying things that have been added since Comfy got a team

### DISCUSSION THREAD

**Discussion by crystalwizard** [comfyanonymous/ComfyUI#5286] [General]
one of the most annoying things that's been added is the new popup when there are errors. instead of just a concise, informative error message, now there's a huge node with large letters that tells you almost nothing, and a link to the github.

**Comment by LukeG89** [comment]
This complaint should be addressed to the frontend repo: https://github.com/Comfy-Org/ComfyUI_frontend
